### DISCUSSION TITLE
[Feature Request] YMod

### DISCUSSION THREAD

**Discussion by TRRESTGHGYTCFTGBCXSZD** [TeamRizu/OutFox#591] [03 - Feature Requests]
### Please Select the game mode your feature request is about.

All Game Modes

### Describe the solution you'd like

YMod is inverse version of XMod, where XMod gets slower when BPM is low, but YMod gets faster when BPM is low.
Arrow Speed Formula For YMod: (YMod/BPM)*YSpeed
For Example:
100Y-2 = 100 YMod, 2 YSpeed
150Y = 150 YMod, 1 YSpeed, 75 BPM = Arrow Moves 2X, 300 BPM = Arrow Moves 0.5X
Special case for Stop/Pause: It does show arrow after that, but not moving

**Comment by ghost** [comment]
this is literally what i've been looking for a while now. we need this.
